src/Installer/redist-installer/projects/SdkResolver.csproj

```diff
@@ -7,11 +7,7 @@
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <GenerateRuntimeConfigurationFiles>false</GenerateRuntimeConfigurationFiles>
     <ExcludeFromSourceOnlyBuild>true</ExcludeFromSourceOnlyBuild>
-    <!--
-      References assets from multiple architectures and can't be built inside the VMR until we have a join point job
-      that can pull assets from multiple legs. https://github.com/dotnet/source-build/issues/4336
-    -->
-    <ExcludeFromDotNetBuild>true</ExcludeFromDotNetBuild>
+    <ExcludeFromDotNetBuild Condition="'$(DotNetBuildPass)' == ''">true</ExcludeFromDotNetBuild>
```

Comment:
Do you think there is likely to be a pass 3 (that isn't the merge pass)? If so, then I think we should be explicit about this being pass 2.

Reply:
I think it depends. If we think of pass 2 as "a pass that needs artifacts from pass 1 on other verticals" and pass 3 as "a pass that needs artifacts from pass 2 on other verticals", there _could_ be scenarios where that's needed, especially around workloads.

Do we _want_ to constrain this to only ever work in pass 2, or simply "not in pass 1, which by definition only has one vertical and therefore won't include the required packages"

Reply:
Yes, we should be explicit and only produce and upload artifacts that are needed in that build pass. I will submit a PR to fine tune this.